src/opers.h

```diff
@@ -235,9 +235,14 @@ extern Obj TRY_NEXT_METHOD;
 **
 **  Note that 'ELM_FLAGS' is a macro, so do not call it  with arguments  that
 **  have side effects.
+**
+**  C_ELM_FLAGS returns a result more convenient for use inside the kernel
+**  since the C compiler can't know that True != False.
```

Comment:
I don't follow: How is that "more convenient"?
Perhaps you meant to say it enables additional optimization opportunities?

Reply:
Yes. That is what I meant. I'll rephrase.